.gitignore

```diff
@@ -0,0 +1,8 @@
+
+
+# File types
```

Comment:
might be nice to add a generated gitignore that includes files like .DS_Store etc.